packages/canary-react/src/stories/ic-time-selector.stories.js

```diff
@@ -0,0 +1,317 @@
+ * The `disableTimes` prop accepts an array of time objects or strings.
+ * Each time object can have a `start` and `end` property to define a range of disabled times, or it can be a single time string to disable that specific time.
+ * The time can be in any format supported as `timeFormat`, in ISO 8601 time string format (`HH:MM:SS`) or as a JavaScript `Date` object.
+ */
+export const DisableTimes = {
+  render: () => (
+      <IcTimeSelector
+        disableTimes={[{ start: "08:00", end: "10:00" }, "13:20"]}
+      />
+),
+  name: "Disable times",
+};
+
+/**
+ * Demonstrates the size options for the time selector component.
+ */
+export const Small = {
+  render: () => (
+    <IcTimeSelector size="small" timePeriod="12" />
+  ),
+  name: "Small",
+};
+
+export const Large = {
+  render: () => (
+    <IcTimeSelector size="large" timePeriod="12" />
+  ),
+  name: "Large",
+};
+
+/**
+ * The time selector can be cleared by setting the the value attribute to one of the following:
+ * - empty string
+ * - `null`
+ * - `undefined`
+ */
+export const WithClearingValue = {
+  render: () => {
+    const [value, setValue] = useState("08:30:00");
+    const handleUpdate = () => {
+      setValue("08:30:00");
+    };
+    const handleClearValue = (value) => {
+      setValue(value);
+    };
+    return (
+      <>
+        <IcTimeSelector
+          value={value}
+        />
+        <br />
+        <div style={{ display: "flex", gap: "8px" }}>
+          <IcButton onClick={() => handleUpdate()}>Update time</IcButton>
+          <IcButton onClick={() => handleClearValue(null)}>Set null</IcButton>
+          <IcButton onClick={() => handleClearValue("")}>
+            Set Empty String
+          </IcButton>
+          <IcButton onClick={() => handleClearValue(undefined)}>
+            Set undefined
+          </IcButton>
+        </div>
+      </>
+    );
+  },
+  name: "With clearing value",
+};
+
+export const UpdatingValue = {
+  render: () => {
+    const [value, setValue] = useState("08:30:00");
+    const handleReset = () => {
+      setValue("08:30:00");
+    };
+    const handleNewValue = () => {
+      setValue("10:45:15");
+    }
+    return (
+      <>
+        <IcTimeSelector
+          value={value}
+        />
+        <br />
+        <div style={{ display: "flex", gap: "8px" }}>
+          <IcButton onClick={() => handleReset()}>Reset time</IcButton>
+          <IcButton onClick={() => handleNewValue()}>Set new time</IcButton>
+        </div>
+      </>
+    );
+  },
+  name: "Updating value",
+};
+
+const defaultArgs = {
+  disableTimes: [],
+  max: "",
+  min: "",
+  size: "medium",
+  theme: "inherit",
+  timeFormat: "HH:MM:SS",
+  timePeriod: "24",
+  value: "12:00:00",
+};
+
+/**
+ * Go to the <ic-link href="/?path=/story/react-components-time-selector--playground">separate page for the playground example</ic-link> to view the prop controls.
+ */
+
+export const Playground = {
```

Comment:
When I set `minTime` in the playground to 09:00:00 and then refresh, the time selector moves to 09:00:00 but this means it overrides the value of the `value` prop:
<img width="188" height="165" alt="image" src="https://github.com/user-attachments/assets/3552bf90-cf84-411d-ba83-f130182b44e5" />
I think the selected time / `value` should always take precedence i.e. it should still display 12:00:00 on load